.prettierignore

```diff
@@ -0,0 +1,2 @@
+packaging/*
```

Comment:
We ignore this, because `conda` files might contain template syntax which is not supported by plain yaml.